### PR TITLE
Android: Add Playstore metadata and helper tools

### PR DESCRIPTION
**Pull request by carstene1ns**

### CODE REVIEW

**Comment by Ghabry** [comment]
Great to have this in the repo now. Can't really review this due to lack of knowledge.
The git changelog generator works excellent.

Maybe the biggest 10" PNG could be replaced with a JPEG because even a 100% quality JPEG is already 200kbyte smaller.

Otherwise can only discuss nitpicky stuff like "I don't like your game selection" ;).

**Comment by carstene1ns** [comment]
Well yeah, these were the games I had on my Tablet, when I needed the screenshots... 📲

I have added some README files to ease the use of the tools.

The `transifex_helper` script was needed, because you cannot upload text files without making them different resources, also no comments possible. So it fakes being a native Android xml. 😁

File: builds/android/transifex_helper.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+# default options
+metadata_dir = "metadata"
+lang_code = "en-US"
+mode = ""
+
+optparse = OptionParser.new do |opts|
+  opts.banner = "Usage: transifex_helper.rb [options]"
+  opts.separator ''
+  opts.on("-m", "--mode MODE", "Set operation mode: split, join", String) do |m|
```
Comment (Ghabry) [inline builds/android/transifex_helper.rb]: The documentation for split and join could be improved. Looks like join takes the txt files and writes them in a XML and split is the inverse operation.

**Comment by Ghabry** [comment]
Okay so we have slots for 10 games.
Based on the play store statistics and would suggest an arbitrary selection of:

3 Generic/Japanese games, 1 Korean, 1 German, 1 Portuguese/Brazilian, 1 Spanish, 1 Russian and 1 French.

There is still one slot free, which language do you want? :D

For the 3 generic I would vote for YN, IB and Wadanohara. Problem with Japanese games is that they are usually known worldwide :D.

Korean: Not sure yet, in the end is probably a Japanese one...

German: Vampires Dawn. I like UiD more but VD is more hyped >.>

Portuguese: Maybe @fdelapena can find a good one.

Spanish: @fdelapena what is a Spanish classic?

French: Aedemphia

Russian: @Demetrius

**Comment by carstene1ns** [comment]
I also vote for IB, YN, VD, OFF and maybe Standstill Girl as classics.
I would like to have some newer games included, but they are unfurtunately not well known… :/

**Comment by Ghabry** [comment]
Here is the bug report telemetry, maybe we should really get an optional usage data telemetry to figure out the most popular games and not the most broken games :P

via `grep -rni . -e "Loading game" -a -m 1  | cut -d' ' -f6- | sort | uniq -c | sort -n -r`

```
     20 Yume Nikki
     20 The Gray Garden
     16 Off
     15 Ib
     14 ドラえもん、のび太のBIOHAZARD
     14 Vampires Dawn 2
     12 Wadanohara and the Great Blue Sea
     12 .flow
     12 ゆめ2っき
     11 ドラえもん、のび太のBIOHAZARD2
     11 Vampires Dawn
     11 서프라이시아
     10 �ｽh�ｽ�ｽ�ｽS�ｽ�ｽ�ｽN�ｽG�ｽX�ｽg�ｽS �ｽ�ｽ�ｽo�ｽX�ｽg�ｽ`�ｽ�ｽ ver.1.30
      9 ジョジョの奇妙な冒険
      8 ドラゴンクエスト　悪霊の勇者
      8 のび太戦記ACE
      8 Ruina　廃都の物語　ver1.21
      8 Ib_1.07
      7 ドラえもん、のび太のBIOHAZARD G Rebirth
      7 회색정원
      6 Dragonball Z: Legend of Z
      6 Doraemon: Nobita's Resident Evil
      5 巴哈姆特
      5 [TimeTraveler]
      5 クミとクマ
      4 イストワール　ver2.03
      4 フィアードライブ Ver1.43
      4 the genesis
      4 Super Mario RPG - The Seven Sages
      4 Rainbow Treasure
      4 Feuer um Mitternacht
      4 �ｽ_�ｽC�ｽﾌ托ｿｽ`�ｽ�ｽ
      4 Continue/Stop/Rise
      4 ‚ä‚ß‚É‚Á‚«
      4 7명째의 스탠드사
      3 ドラえもん、のび太のBIOHAZARD EasyType
      3 ふしぎの城のヘレン
      3 VHゲーム０１_161029_パッチ
      3 Untitled
      3 Unterwegs in Düsterburg
      3 The Huntress of the Hollow
      3 Sternenkind-Saga
      3 Pom Gets Wi-Fi
      3 Paranormal Syndrome
      3 Oneshot
      3 DB
      3 Dark Sphere5
      3 Amnesia v1.2
      3 さよなら、モナーＲＰＧ。
      2 すでに私たちは地獄のまっただ中でした。
      2 深沉之憶2
      2 海賊高校生
      2 ドラえもん、のび太のBIOHAZARD(ﾘﾏｽﾀｰ版)
      2 ドラゴンクエストⅣ外伝 ver1.26b EASY TYPE
      2 ドラゴンクエスト列伝
      2 Yume 2kki
      2 Yanderella
      2 Wolfenhain (F4 Vollbild)
      2 Wadanohara e o Grande Mar Azul
      2 VHゲーム０１_170404_パッチ
      2 VHゲーム０１_170122
      2 VHゲーム０１_161018_パッチ
      2 VHゲーム０１_160521_パッチ
      2 Space Funeral
      2 O Ataque dos Evils
      2 Nepheshel Ver2.03
      2 Mother: Cognitive Dissonance
      2 Moner special
      2 Long Long Ago
      2 Live Together
      2 Knight Night
      2 JoJo's Bizarre Adventure
      2 ¯‚ÌŠC‚Å
      2 Ib_1.05
      2 Fantasy of dragon saga
      2 Eternal Legends: The Demon Saga
      2 DRAGON FANTASY�ｽU UOpatch
      2 DRAGON FANTASYⅡ UOpatch
      2 Dragon Blade
      2 Dragon ball Futuros alternativos
      2 DQ1-2
      2 DBGT O Retorno do Inferno 2
      2 COSMIC WORLD
      2 도라에몽, 노비타의 BIOHAZARD
      2 A New HOME
      2 Ahriman's Prophecy
      2 パーフェリオンリンク　トゥルース
      2 포켓 몬스터
      1 里見くえすと
      1 †　メイジの転生録
      1 赤ずきんダークサイド
      1 メイドイン誰
      1 ハンター見習いの授業ノート
      1 葬
      1 艾特蘭
      1 神話戰士-復仇的前奏曲 Ver1.20
      1 王様 Quest
      1 あすなな　-アストリア王国騎士団第七小隊-
      1 ﾊｲﾚﾍﾞﾙﾊﾞｰｼﾞｮﾝ ﾃﾞｨﾚｸﾀｰｽﾞｶｯﾄ版 ver2.24
      1 深沉之憶3
      1 海の向こう 闇空を越えて
      1 正式名称未定
      1 東方遊演戯 ver2.0
      1 東方星母続録
      1 東方Untitled
      1 東方ライブアライブ
      1 新訳　ドラえもん のび太のBIOHAZARD
      1 もしも ほぼみんな 敵だったら
      1 支配の指輪
      1 捞摹肺 家斥 扁藕
      1 戦闘戦闘また戦闘　-もっと、もっとだ！編-
      1 シルフェイド幻想譚
      1 平偉本色V1.9D
      1 いちろ少年忌譚
      1 のび太戦記　～導かれし者達～
      1 ドラえもん、のび太のBIOHAZARD β版+Mode
      1 ドラえもん、のび太のBIOHAZARD OUTBREAK
      1 ドラえもん、のび太のBIOHAZARD G
      1 大海原与大海原 ver1.03【丝绮拉工作室汉化】
      1 夢日誌
      1 器南 阁胶磐
      1 勇者伝説
      1 フランドールの勇者
      1 ウォーターは俺の娘
      1 もえ〜伝
      1 あなたと三将軍のラブラブデート
      1 タオルケットをもう一度「１」
      1 �ｽW�ｽ�ｽ�ｽW�ｽ�ｽ�ｽﾌ奇妙�ｽﾈ冒�ｽ�ｽ
      1 VIPRPG紅白2015 No.37 もしもンクエストモンスターズ2 ～ポテチのふしぎな鍵～ β20151129
      1 VIPRPGGW2017 No.4 もしもンクエストモンスターズ2 ～ポテチのふしぎな鍵～ β20170508
      1 VHƒQ[ƒ€‚O‚P_160521_ƒpƒbƒ`
      1 VHゲーム０１_170530
      1 Vampire X Knight - Episode 01
      1 Unterwegs in Duesterburg
      1 Touhou Seiboroku ~ Touhou Mother
      1 Tod auf Reisen
      1 The Maid of Fairewell Heights
      1 The Dark Side of Red Riding Hood
      1 The Damnation
      1 Teitai Shoujo
      1 star wars
      1 Space Funeral: EARTH BIRTH
      1 Single 던전&파이터
      1 Secret of Nahsa
      1 Rygar no Reino Perdido
      1 RPG Heroes
      1 Rosy Despair
      1 Resident Evil Demake
      1 キラーナイツ－R
      1 Princess_Heart -Full Power Edition-
      1 Pom Gets Wi-Fi 1-04 BR
      1 Polka
      1 Pokémon Dusk
      1 Pokémon Dawn
      1 Phoenix
      1 Paranormal Syndrome 2
      1 OUDOU QUESTⅡ
      1 OUDOU QUEST
      1 Nobihaza EscapeIsland
      1 Mystic Star
      1 MONSTER×MONSTER
      1 MONIGOTE FANTASY
      1 Mondschein
      1 Mondlichts Kinder V2
      1 Makoto Mobius
      1 Lost Memory
      1 Legion Saga
      1 La Vie d'un PNJ
      1 La Escalera
      1 さいはてHOSPITAL
      1 Hero's Realm
      1 Gromada
      1 Gate of Epsilon
      1 Garden Gray The
      1 Final Fantasy- Endless Nova
      1 Final Fantasy Blackmoon Prophecy Plus
      1 FINAL FAMTASY
      1 En-Sof
      1 END ROLL
      1 Elektra Kingdom
      1 El caballero
      1 DRAGON QUEST COOL
      1 DRAGON FANTASYⅡ DQFFpatch
      1 DRAGON FANTASY 1.5
      1 DRAGON FANTASYⅡ
      1 Dragon Destiny by Rast (x_rast_x@hotmail.com : http://www.rastblackwing.com)
      1 Dragon Ball Z - A Vingança Da Raça Tsufu
      1 Dragon Ball Collection
      1 DQP
      1 Dhux's Scar
      1 DevilBusters
      1 death house the Last Nightmare
      1 DBGT Universo Paralelo 2
      1 DBGT O Retorno do Inferno
      1 DBAF
      1 Dark Magic
      1 CDZ - A Batalha dos Deuses
      1 Cavaleiros Hades
      1 Brave Fantasia
      1 Blue Skies
      1 As Duas Sagas !
      1 Ara Fell
      1 Alter A.I.L.A. Genesis
      1 괴이증
```

**Comment by Ghabry** [comment]
Based on twitter feedback and personal taste I made a collection of 10 games. The ZIP contains a fake RPG Maker Game structure, enough for the app to detect them.

As Twitter game relevance criteria I used "has a Lets Play on Youtube".

ZIP with a layout suitable for the App to show title screens.

[rpgmaker.zip](https://github.com/EasyRPG/Player/files/1965275/rpgmaker.zip)

 - Celestial Silfate Story (Japanese, via Twitter)
 - Ib (Worldwide / Japanese)
 - Kindred Saga (via Twitter, English/Japanese???)
 - La Cicatriz de Dhux (Spanish, Used the international title screen, the Spanish has creepy eyes) 
 - Ruina (popular in China)
 - The Way (via Twitter, English)
 - Unterwegs in Düsterburg (German)
 - Velsarbor (German/French)
 - Witch's Heart (personal preference, worldwide)
 - Yume Nikki (Worldwide / Japanese)

montage

**Comment by Ghabry** [comment]
Proposed new three screenshots:

Phone:

phonescreenshots

seven inch:

seveninchscreenshots

ten inch:
teninchscreenshots

@carstene1ns 

Are they ago? Then we can (finally) merge thsi

**Comment by carstene1ns** [comment]
Will include them when I am home later. Looks good to me.

**Pull request merged**